include/bitcoin/explorer/command.hpp

```diff
@@ -702,6 +702,14 @@ class BCX_API command
         setting_.server.client_private_key = value;
     }
 
+    /**
+     * Get the bitcoin settings.
+     */
+    virtual bc::settings& get_bitcoin_settings()
```

Comment:
This is generated source. Each individual setting is generated from metadata in an xml file. The design here will need to be different.

Reply:
I see. I didn't pay attention to the comment above. I'm not familiar with gsl at all but I'll try to figure out the required changes by going through the existing gsl code in the model directory. @evoskuil please let me know if there's something important I would need to know when working with gsl.